### PR TITLE
intl: remove invalidated or removed translations

### PR DESCRIPTION
**Pull request by sjd78**
After pushing base English strings to zanata for ovirt 4.5, do an initial pull to:
  - remove keys the have been removed
  - remove keys for translations that have been invalidated by changes to the English text

### CODE REVIEW

**Comment by ovirt-infra** [comment]
All tests passed

File: src/intl/translated-messages.json [hunk 1]
```diff
@@ -639,10 +638,8 @@
     "utilizationCardDiskUsed": "{used} von {total} {storageUnits} verwendet",
     "utilizationCardLegendAvailable": "Verfügbar",
     "utilizationCardLegendAvailableP": "% verfügbar",
-    "utilizationCardLegendAvailableWithDetails": "Verfügbar {value}% ({diskPath})",
```
Comment (rszwajko) [inline src/intl/translated-messages.json:642]: Please re-apply https://github.com/oVirt/ovirt-web-ui/pull/1541
Translations for 2 labels were manually fixed there:
1. `utilizationCardLegendAvailableWithDetails`
2. `utilizationCardLegendUsedWithDetails`

**Review by rszwajko — CHANGES_REQUESTED** [review]

File: src/intl/translated-messages.json (hunk 1 above)
Comment (rszwajko) [reply src/intl/translated-messages.json:642]: Fixed in Zanata

**Review by rszwajko — APPROVED** [review]

File: src/intl/translated-messages.json (hunk 1 above)
Comment (sjd78) [reply src/intl/translated-messages.json:642]: I'll do another pull and update the PR.  :smile:

**Comment by ovirt-infra** [comment]
All tests passed

File: src/intl/translated-messages.json [hunk 2]
```diff
@@ -1184,10 +1182,10 @@
     "utilizationCardDiskUsed": "{used} de {total} {storageUnits} usadas",
     "utilizationCardLegendAvailable": "Disponible",
     "utilizationCardLegendAvailableP": "% disponible",
-    "utilizationCardLegendAvailableWithDetails": "Disponible {value}% ({diskPath})",
```
Comment (rszwajko) [inline src/intl/translated-messages.json:1187]: Incorrect language used.

File: src/intl/translated-messages.json [hunk 3]
```diff
@@ -1729,10 +1726,8 @@
     "utilizationCardDiskUsed": "{used} parmi {total} {storageUnits} utilisés",
     "utilizationCardLegendAvailable": "Disponible",
     "utilizationCardLegendAvailableP": "% disponible",
-    "utilizationCardLegendAvailableWithDetails": "Disponible {value}% ({diskPath})",
```
Comment (rszwajko) [inline src/intl/translated-messages.json:1732]: Missing fix in Zanata

File: src/intl/translated-messages.json [hunk 4]
```diff
@@ -1866,9 +1861,7 @@
     "utilizationCardAvailable": "Disponibile",
     "utilizationCardLegendAvailable": "Disponibile",
     "utilizationCardLegendAvailableP": "% Disponibile",
-    "utilizationCardLegendAvailableWithDetails": "Disponibile {value}% ({diskPath})",
```
Comment (rszwajko) [inline src/intl/translated-messages.json:1869]: Missing fix in Zanata.

**Review by rszwajko — CHANGES_REQUESTED** [review]

File: src/intl/translated-messages.json (hunk 2 above)
Comment (rszwajko) [reply src/intl/translated-messages.json:1187]: Fixed in Zanata

File: src/intl/translated-messages.json (hunk 4 above)
Comment (rszwajko) [reply src/intl/translated-messages.json:1869]: Fixed in Zanata

File: src/intl/translated-messages.json (hunk 3 above)
Comment (rszwajko) [reply src/intl/translated-messages.json:1732]: Fixed in Zanata

**Comment by rszwajko** [comment]
Note that #1582  has been created for adding Georgian language.

**Comment by ovirt-infra** [comment]
All tests passed

File: src/intl/translated-messages.json [hunk 5]
```diff
@@ -1732,7 +1729,7 @@
     "utilizationCardLegendAvailableWithDetails": "Disponible {value}% ({diskPath})",
     "utilizationCardLegendUsed": "Utilisé",
     "utilizationCardLegendUsedP": "% utilisé",
-    "utilizationCardLegendUsedWithDetails": "Utilisé {value}% ({diskPath})",
```
Comment (rszwajko) [inline src/intl/translated-messages.json:1735]: A minor issue: missing right bracket( just fixed in Zanata)

**Review by rszwajko — APPROVED** [review]

**Pull request merged**